src/components/CommentList.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import { Comment } from './Comment';
+
+export const CommentList = ({ comments, postId }: CommentsProps) => {
+  return (
+    <>
+      <p className="comment__heading">Comments</p>
+      {comments.map((item) => <Comment key={item.id} {...item} postId={postId} />)}
```

Comment:
```suggestion
      {comments.map((item) => (
        <Comment key={item.id} {...item} postId={postId} />
      ))}
```
don't write everything in one line

Reply:
✅